pristine-zen-garden.html

```diff
@@ -0,0 +1,198 @@
+        <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Check the validity of this site&#8217;s CSS" class="zen-validate-css">CSS</a>
+        <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Creative Commons license of this site: Attribution-NonCommercial-ShareAlike." class="zen-license">CC</a>
+        <a href="http://mezzoblue.com/zengarden/faq/#aaa" title="Read about the accessibility of this site" class="zen-accessibility">A11y</a>
+        <a href="https://github.com/mezzoblue/csszengarden.com" title="Fork this site on Github" class="zen-github">GH</a>
+      </footer>
+    </div>
+
+    <aside class="sidebar" role="complementary">
+      <div class="wrapper">
+        <div class="design-selection" id="design-selection">
+          <h3 class="select">Select a Design:</h3>
+          <nav role="navigation">
+            <ul>
+              <li><a href="/205/" class="design-name">spring360</a> by <a href="http://www.medialab360.com" class="designer-name">Rene Hornig</a></li>         
+              <li><a href="/204/" class="design-name">Withering Beauty</a> by <a href="http://www.wduffy.co.uk/" class="designer-name">William Duffy</a></li>         
+              <li><a href="/203/" class="design-name">Tiny Blue</a> by <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a></li>         
+              <li><a href="/202/" class="design-name">Retro Theater</a> by <a href="#" class="designer-name">Eric Rogé</a></li>         
+              <li><a href="/201/" class="design-name">Lily Pond</a> by <a href="#" class="designer-name">Rose Thorogood</a></li>         
+              <li><a href="/200/" class="design-name">Icicle Outback</a> by <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a></li>         
+              <li><a href="/199/" class="design-name">CSS ZEN ARMY</a> by <a href="http://www.niceguy.com" class="designer-name">Carl Desmond</a></li>         
+              <li><a href="/198/" class="design-name">The Original</a> by <a href="#" class="designer-name">Joachim Shotter</a></li>         
+            </ul>
+          </nav>
+        </div>
+
+        <div class="design-archives" id="design-archives">
+          <h3 class="archives">Archives:</h3>
+          <nav role="navigation">
+            <ul>
+              <li class="next"><a href="/214/page3"> Next Designs <span class="indicator">&rsaquo;</span></a></li>
+              <li class="previous"><a href="/214/page1/"><span class="indicator">&lsaquo;</span> Previous Designs </a></li>
+              <li class="viewall"><a href="http://www.mezzoblue.com/zengarden/alldesigns/" title="View every submission to the Zen Garden."> View All Designs </a></li>
+            </ul>
+          </nav>
+        </div>
+
+        <div class="zen-resources" id="zen-resources">
+          <h3 class="resources">Resources:</h3>
+          <ul>
```

Comment:
A few front and back spacing missing in this <ul> to be consistent with the indentation choice for <li> text. It does help the reading - but it need to go smoothly with the CSS choices (could be misleading with padding etc...)

Reply:
Thanks yea I guess those line breaks are pretty inconsistent throughout the document, I was trying to find a simple consistent rule that didn't add too many empty lines, will have another go

I'm not sure I understand about lines 161- 18X, do you mean an empty line on either side of the <ul> element? I can't figure out how to quote a possible change but something like:

<div>
  <h3>
  
  <ul>
    ...
  </ul>

</div>?

Reply:
Ah no I wrote raw html into the comment and it parsed it

Reply:
What I'm trying to ask is do you think there are inconsistent line breaks or inconsistent indentation with that last ul element?

Reply:
What I meant was you have just an inconsistent spacing I think like here you get ;
+              <li class="next"><a href="/214/page3"> Next Designs <span class="indicator">&rsaquo;</span></a></li>
+              <li class="previous"><a href="/214/page1/"><span class="indicator">&lsaquo;</span> Previous Designs </a></li>
+              <li class="viewall"><a href="http://www.mezzoblue.com/zengarden/alldesigns/" title="View every submission to the Zen Garden."> View All Designs </a></li>

with spaces like \_View All Designs_

But before your list goes as ;
 +              <li><a href="/205/" class="design-name">spring360</a> by <a href="http://www.medialab360.com" class="designer-name">Rene Hornig</a></li>         
+              <li><a href="/204/" class="design-name">Withering Beauty</a> by <a href="http://www.wduffy.co.uk/" class="designer-name">William Duffy</a></li>         
+              <li><a href="/203/" class="design-name">Tiny Blue</a> by <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a></li>         
+              <li><a href="/202/" class="design-name">Retro Theater</a> by <a href="#" class="designer-name">Eric Rogé</a></li>         
+              <li><a href="/201/" class="design-name">Lily Pond</a> by <a href="#" class="designer-name">Rose Thorogood</a></li>         
+              <li><a href="/200/" class="design-name">Icicle Outback</a> by <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a></li>         
+              <li><a href="/199/" class="design-name">CSS ZEN ARMY</a> by <a href="http://www.niceguy.com" class="designer-name">Carl Desmond</a></li>         
+              <li><a href="/198/" class="design-name">The Original</a> by <a href="#" class="designer-name">Joachim Shotter</a></li>  

With no space in between the tag \<li> and the printed text. 

What I was trying to say is that the spacing in the first writing help the reading of the file, I think it is good, but at the same time adding the blank space in the HTML text can be a pain for a further styling in CSS.


It is definitively just to make the code squeaky clean because, it seems all perfect to me on all the rest.

Reply:
Ah I see! Totally missed it, I'll just make them the same...

Reply:
How's that?

Reply:
Perfect -I'll merge them together :)